firefox-ios/Client/Frontend/Home/HomepageViewController.swift

```diff
@@ -258,6 +258,8 @@ class HomepageViewController:
         collectionView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
         collectionView.backgroundColor = .clear
         collectionView.accessibilityIdentifier = a11y.collectionView
+        let interaction = UIContextMenuInteraction(delegate: self)
+        collectionView.addInteraction(interaction)
```

Comment:
```suggestion
        collectionView.addInteraction(UIContextMenuInteraction(delegate: self))
```
nit: assignment don't add much or really improve legibility or line length here.